Week3/homework/step2-3.js

```diff
@@ -31,8 +35,13 @@ function repeatStringNumTimesWithDoWhile(str, num) {
   // eslint-disable-next-line
   let result = '';
 
-  // Replace this comment and the next line with your code
-  console.log(str, num, result);
+  let i = 0;
+  if (num > 0) {
```

Comment:
👍 for checking if the `do-while` loop should be entered at all. Because the `do-while` loop does not check the loop condition upfront, it is unsafe to use. It almost always better to use a regular `while` loop that checks the loop condition before entering the loop body. The purpose of this exercise was to demonstrate this.